packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/guides/Migration.tool.md

```diff
@@ -0,0 +1,522 @@
+
+### Upgrading Webpack
+
+If you are already using @cerner/webpack-config-terra ^1.0.0 you can skip this section.
+
+A complete upgrade guide can be found [here](https://engineering.cerner.com/terra-ui/dev_tools/cerner/webpack-config-terra/upgrade-guide). Webpack can be upgraded separately from terra-functional-testing. This change does not need to be bundled with test updates, but it does need to happen before the test updates.
+
+Install @cerner/webpack-config-terra, @cerner/terra-aggregate-translations, and postcss ^8.2.1 (a required peer dependency of webpack-config-terra):
+
+```sh
+npm install --save-dev @cerner/webpack-config-terra @cerner/terra-aggregate-translations postcss@8.2.1
+```
+
+If your project utilizes the `tt-serve-static` bin command go ahead and install [terra-cli](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-cli/about) as you'll need it to upgrade the static asset utilities:
+
+```sh
+npm install --save-dev @cerner/terra-cli
+```
+
+Update any references to the old webpack configuration:
+
+```diff
+// webpack.config.js
+- const webpackConfig = require('terra-dev-site/config/webpack/webpack.config');
++ const webpackConfig = require('@cerner/webpack-config-terra');
+```
+
+Update any references to the old terra-aggregate-translations:
+
+
+```diff
+// jestGlobalSetup.js
+- const aggregateTranslations = require('terra-aggregate-translations');
++ const aggregateTranslations = require('@cerner/terra-aggregate-translations');
+```
+
+Update any scripts in the package.json to use webpack-dev-server. Use terra-cli for any static sites. Remove `terra-aggregate-translations` if installed:
+
+```diff
+// package.json
+{
+  "scripts": {
+-    "start": "tt-serve",
+-    "start-prod": "tt-serve --env.disableHotReloading -p",
+-    "start-static": "npm run pack && tt-serve-static",
++    "start": "webpack-dev-server",
++    "start-prod": "webpack-dev-server --env.disableHotReloading -p",
++    "start-static": "npm run pack && terra express-server --site ./build",
+  },
+  "devDependencies": {
++   "@cerner/terra-aggregate-translations": "^2.0.1",
++   "@cerner/terra-cli": "^1.3.0",
++   "@cerner/webpack-config-terra": "^1.2.0",
++   "postcss": "^8.2.1",
+-   "terra-aggregate-translations": "^1.0.0"
+  }
+}
+```
+
+At this point it is recommended to do a clean install prior to testing each of the scripts. If you have the script go ahead and run `npm run clean:install`. Alternatively ensure that the old node_modules directory is deleted and reinstall all dependencies.
+
+```sh
+npm run clean:install
+```
+
+Verify each of the scripts are working as intended:
+
+```sh
+npm run start
+
+npm run start-prod
+
+npm run start-static
+```
+
+Note: If you don't have some of these commands it is not necessary to add them. Just ensure the commands you do have are working as expected. All of the `tt-serve` commands should be updated to use webpack-dev-server or the terra express server
+
+### Upgrading ESlint Config Terra
```

Comment:
we could have this section upgrade to eslint 7, 🤷‍♂️

Reply:
Added v7 as a suggestion in the diff when upgrading eslint-config-terra:

https://github.com/cerner/terra-toolkit/pull/574/commits/42750ce4526f2f3d447820f56d3dd5b0f3b8bfc2